### PR TITLE
Add User class example

### PR DESCRIPTION
**Pull request by Mogomotsi-DocFox**

### CODE REVIEW

File: app/example_user.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class User
+    attr_accessor :first_name, :last_name
+    attr_accessor :name, :email
```
Comment (c13e) [inline app/example_user.rb:3]: we use 2-space indentation for both Ruby & JS, you should be able to set it in your text editor but let me know if you need a hand :)

File: app/example_user.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class User
```
Comment (c13e) [inline app/example_user.rb:1]: Just checking if this file should be at `app/example_user.rb`? This looks like a class representing a domain model, and these live under `app/models` in Rails

**Review by c13e — APPROVED** [review]
I think you accidentally committed a bunch of extra files that are empty?

File: app/example_user.rb (hunk 2 above)
Comment (Mogomotsi-DocFox) [reply app/example_user.rb:1]: or this was an example model, was meant to remove it.

File: app/example_user.rb (hunk 1 above)
Comment (Mogomotsi-DocFox) [reply app/example_user.rb:3]: I've configured VSCode to do this, seems like it did not work will have a look at it again.

**Pull request merged**